frontend/src/metabase/ui/components/inputs/MultiSelect/MultiSelect.styled.tsx

```diff
@@ -109,7 +109,7 @@ export const getMultiSelectOverrides =
           backgroundColor: "var(--mb-color-background-selected)",
         },
         defaultValueRemove: {
-          color: theme.fn.themeColor("brand"),
+          color: "var(--mb-color-text-selected)",
```

Comment:
defaultValueRemove is the color for the selected `MulitiSelect` value's cross icon.
![image](https://github.com/user-attachments/assets/387ae933-51ea-42b1-bd9d-dbf839dee53e)